src/client/game.cpp

```diff
@@ -2298,18 +2296,19 @@ void Game::toggleFog()
 
 void Game::toggleDebug()
 {
+	LocalPlayer *player = client->getEnv().getLocalPlayer();
+	bool has_basic_debug = player->hud_flags & HUD_FLAG_DEBUG;
 	// Initial: No debug info
 	// 1x toggle: Debug text
 	// 2x toggle: Debug text with profiler graph
-	// 3x toggle: Debug text and wireframe (needs "debug" priv)
+	// 3x toggle: Debug text and wireframe
```

Comment:
Still needs debug priv

Reply:
Yeah, must've left that change in when ripping out the debug priv removal...